src/server/meta_store.cpp

```diff
@@ -72,18 +84,38 @@ ::dsn::error_code meta_store::get_value_from_meta_cf(rocksdb::DB *db,
                                                      uint64_t *value)
 {
     std::string data;
+    auto ec = get_string_value_from_meta_cf(db, cf, read_flushed_data, key, &data);
+    if (ec != ::dsn::ERR_OK) {
+        return ec;
+    }
+    dassert_f(dsn::buf2uint64(data, *value),
+              "rocksdb {} get {} from meta column family got error value {}",
```

Comment:
```suggestion
              "rocksdb {} get \"{}\" from meta column family failed to parse into uint64",
```

I think you cannot simply print a value out, there could be unprintable characters.